university-agent/pom.xml

```diff
@@ -100,7 +101,16 @@
             <version>2.5</version>
         </dependency>
 
-
+        <dependency>
+            <groupId>org.hibernate</groupId>
```

Comment:
Let's bump spring-boot to the newest version and then this can be removed. Probably.